client/app/directives/contextMenu.js

```diff
@@ -45,6 +45,19 @@
         };
 
         return {
+            restrict: "A",
```

Comment:
I read about these magical letters in AngularJs. 
Is it possible to avoid using these letter ? or it is sort of best practice ?